operations/flags.go

```diff
@@ -26,6 +26,7 @@ const (
 	messageFlagName       = "message"
 	activeFlagName        = "active"
 	refFlagName           = "ref"
+	workingChangesFlag    = "include-working"
```

Comment:
Maybe "uncommitted" with a short form of "u". "working" is ambiguous here because git has a concept of a working tree and a staging area. Uncommitted applies to both.

Reply:
renamed.